flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/runtime/aggregate/MaxAggregate.scala

```diff
@@ -48,8 +57,16 @@ abstract class MaxAggregate[T](implicit ord: Ordering[T]) extends Aggregate[T] {
   override def merge(intermediate: Row, buffer: Row): Unit = {
```

Comment:
Yes, you are right. The cases where `partialValue == null` should be ignored directly.